docs: Add PSA about balenaOS version breakage (#79)
balena-io/open-balena: README.md

---

OpenBalena is a platform to deploy and manage connected devices. Devices run

> [balenaOS][balena-os], a host operating system designed for running

Reviewer: No reason for this change -- `balena-os` links to the repo but given this is a high level overview it makes sense to link to the marketing site, which `balena-os-website` does.

To learn more about openBalena, visit [balena.io/open][open-balena-website].

> > **IMPORTANT:** Due to changes in [balenaOS], only versions up to and including `2.49.0` are currently supported.

Reviewer: There's no `balenaOS` link below, but there's `balena-os`, so you could:

```suggestion
> **IMPORTANT:** Due to changes in [balenaOS][balena-os], only versions up to and including `2.49.0` are currently supported.
```

> > **IMPORTANT:** Due to changes in [balenaOS][balena-os], only versions up to and including `2.49.0` are currently supported.

Reviewer: I also think it makes sense to move this down below on the "Getting started" section.

> > **IMPORTANT:** Due to changes in the [balena-supervisor] codebase, only balenaOS versions below `2.49.0` are currently supported.

Reviewer: ```suggestion
> **IMPORTANT:** Due to changes in balenaOS, only versions up to and including `2.49.0` are currently supported.
```

Author: agreed, better.

[open-balena-vpn]: https://github.com/balena-io/open-balena-vpn
[open-balena-website]: https://balena.io/open
[pulls]: https://github.com/balena-io/open-balena/pulls

> [balena-supervisor]: https://github.com/balena-io/balena-supervisor

Reviewer: This isn't needed anymore.
